pontoon/messaging/static/css/messaging.css

```
@@ -109,4 +136,115 @@
       width: 285px;
     }
   }
+
+  #review {
+    p {
+      color: var(--light-grey-6);
+      font-size: 16px;
+      line-height: 1.5;
+    }
+
+    p.transactional {
+      margin-top: 20px;
+      font-style: italic;
+      padding: 20px;
+      background: var(--background-1);
+      border-radius: 5px;
+
+      i {
+        color: var(--status-translated);
+      }
+    }
+
+    .body {
+      .value {
+        h1,
+        h2,
+        h3,
+        h4,
+        h5,
+        h6 {
+          color: inherit;
+          font-size: inherit;
+          font-style: inherit;
+          font-weight: bold;
+          letter-spacing: inherit;
+          margin: inherit;
+          padding-bottom: 1em;
+          text-transform: inherit;
+        }
+
+        h1 {
+          font-size: 2em;
+        }
+
+        h2 {
+          font-size: 1.8em;
+        }
+
+        h3 {
+          font-size: 1.5em;
+        }
+
+        h4 {
+          font-size: 1.3em;
+        }
+
+        h5 {
+          font-size: 1.15em;
+        }
+
+        p {
+          padding-bottom: 1em;
+
+          a {
+            color: var(--status-translated);
+            font-weight: inherit;
+          }
+        }
+
+        ol,
+        ul {
+          color: var(--light-grey-6);
+          font-size: 16px;
+          margin-left: 1.5em;
+          padding-bottom: 1em;
+        }
+
+        ul {
+          overflow: inherit;
+          list-style: disc;
+        }
+
+        li {
+          color: var(--light-grey-6);
+        }
```

```suggestion
        li {
          color: var(--light-grey-6);
          a {
            color: var(--status-translated);
            font-weight: inherit;
          }
        }
```

Let's use the same formatting for links in lists.

---

Pushing changes to stage. Now styling for links doesn't only apply to links within paragraphs, but in general.